packages/core/src/standards/helpers.ts

```diff
@@ -14,6 +14,15 @@ export function bufferSourceToArray(chunk: BufferSource): Uint8Array {
   }
 }
 
+export function bufferToArrayBuffer(buffer: Buffer): ArrayBuffer {
```

Comment:
I couldn't find where you're otherwise converting Data modules to ArrayBuffers, so I made this helper. Happy to replace it with however else you're converting these.

Reply:
aha til! thanks.